app/scripts/modules/core/src/deploymentStrategy/strategies/redblack/AdditionalFields.tsx

```diff
@@ -24,7 +24,7 @@ export class AdditionalFields extends React.Component<IRedBlackStrategyAdditiona
           <label>
             <input
               type="checkbox"
-              checked={command.rollback.onFailure}
+              checked={command.rollback?.onFailure}
```

Comment:
not essential, but might be worth ensuring this prop is always a boolean so the componet is more clearly controlled (vs uncontrolled)
```suggestion
              checked={command.rollback?.onFailure ?? false}
```

Reply:
Good call!